kong/db/declarative/import.lua

```diff
@@ -252,11 +311,18 @@ local function load_into_cache(entities, meta, hash)
       assert(type(ws_id) == "string")
 
       local cache_key = dao:cache_key(id, nil, nil, nil, nil, item.ws_id)
+      -- only do transformations if have definitions
```

Comment:
```suggestion
```

Feels like self explanatory (and the english is not right here, so let's remove this).